src/query/service/src/pipelines/processors/transforms/hash_join/result_blocks.rs

```diff
@@ -628,6 +629,11 @@ impl JoinHashTable {
     pub(crate) fn set_validity(column: &ColumnRef, validity: &Bitmap) -> Result<ColumnRef> {
         if column.is_null() {
             Ok(column.clone())
+        } else if column.is_const() {
+            let col: &ConstColumn = Series::check_get(column)?;
+            let validity = validity.clone();
+            let inner = Self::set_validity( col.inner(), &validity.slice(0, 1))?;
```

Comment:
```suggestion
            let inner = Self::set_validity(col.inner(), &validity.slice(0, 1))?;
```

Seems we need a format.